### DISCUSSION TITLE
Minute Problem in DateTimePicker

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#4184] [Help] [answered]
Hi All,

I have a problem when displaying minute in DateTimePicker,
but when I use placeholder, it give me correct result.
my timezone in app.config is UTC
`'timezone' => 'UTC',`

I only have just 1 record in my table.
![image](https://user-images.githubusercontent.com/6214964/192113350-a98e66fd-03b5-4644-84e8-5919673f3136.png)

at display :
![image](https://user-images.githubusercontent.com/6214964/192113189-9fdae02e-684d-41c5-8ee4-7c71609f7d8b.png)

source code from resource file
```
\Filament\Forms\Components\DateTimePicker::make('created_at')
    ->label('Created At')
    ->displayFormat('d-M-Y H:i:s')
    ->disabled(),

\Filament\Forms\Components\Placeholder::make('placeholder1')
    ->label('Created At')
    ->content(
        function (closure $get) {
            $dept = Department::find(1);
            return $dept->created_at->format('d-M-Y H:i:s');
        }
    ),
```

I tried using $model in another placeholder, and it give correct result .
```
\Filament\Forms\Components\Placeholder::make('placeholder2')
    //->label('Created At')
    ->content(
        function ($record) {
            //dd ($record);
            return $record->created_at->format('d-M-Y H:i:s');
        }
    ),
```

How to fixing result from DateTimePicker ?
Thank you.

**Comment by cdburgess** [comment, marked as answer]
In your closure, I see that you are using the Dept model. However, there is no indication on the database record image that it's the same table. I know this might be a stretch, but are you referencing the same model on both form fields? Meaning, is the resource you are using referencing the Dept table? Since there are no references to the database record NOR the model the resource is accessing, it may be difficult to know help troubleshoot.

The other thing I would note is `closure` doesn't look correct. Do you mean `callable`? It doesn't look like you are even using it so you don't need the `closure $get`, you can leave it blank. I ran the following:
```
Forms\Components\DatePicker::make('created_at')
      ->label('Created At')
       ->displayFormat('d-M-Y H:i:s')
       ->disabled(),
 Forms\Components\Placeholder::make('placeholder1')
       ->label('Created At')
       ->content( function () {
              $dept = Dept::find(1);
              return $dept->created_at->format('d-M-Y H:i:s');
       } ),
```
and everything worked as expected.
![Screen Shot 2022-09-24 at 9 05 03 PM](https://user-images.githubusercontent.com/216284/192126386-10a9fb87-7b59-400a-9490-d8a0ad39533a.png)

**Reply by jigujigu** [reply]
Hi Chuck Burgess,
Thank you for your help.

Yes,.. I'm using same table and same record.

I tried to modify the code, like below. 
I used $record and remove the format statment.
Hope it make more clear.

```
\Filament\Forms\Components\DateTimePicker::make('created_at')
    ->label('Created At')
    ->disabled(),

\Filament\Forms\Components\TextInput::make('created_at_temp')
    ->label('Created At')
    ->placeholder(
        fn ($record) => ($record?->created_at)
    )
    ->disabled(),
```

The TextInput & Placehlder give me correct result, And Since this field is readonly, I use this temporary solution for solve my problem.
But I'm still wondering. why DateTimePicker give me worng result ?.

![image](https://user-images.githubusercontent.com/6214964/192168661-7ef7e3cb-6bcf-4ba2-b73c-c07f378d2d76.png)

This is only a recond in my department table.
![image](https://user-images.githubusercontent.com/6214964/192167688-cbc6e8a6-b090-475f-9422-0fd5cc0f5ea5.png)

**I  just suspect than datetimepicker not showing a minutes value, but a numeric of a month.**
This is base on what I trial.  I changed the month in my laptop (os system date) to october, november, december,january and soon, then I  created and saved the data. And I also tried to update create_at value directly at database.
But the minutes in datetimepicker, showing  following the numeric of a month. 10,11,12, 01  and soon.

Any other Ide or the  way  to  the debuging ?

**Comment by cdburgess** [comment]
That is certainly strange. I can't even replicate so I'm not sure where to even start looking to try to help.
